x11-libs/vte/files/vte-0.70-integer-value-is-outside-the-valid-range-clang-16.patch

```diff
@@ -0,0 +1,47 @@
+# From 9400d41660e6c27b672f9c77855d736581f499bf Mon Sep 17 00:00:00 2001
```

Comment:
Thanks, lgtm now. I personally put our own notes above the "From" line (so it doesn't look like they wrote it), but that's personal taste.

Cheers!

Reply:
Nah, lets keep it that way, after all Khem is the original author.